packages/cloud-cognitive/src/components/CreateSidePanel/_create-side-panel.scss

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright IBM Corp. 2021, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/import-once/scss/import-once';
+@import '../../global/styles/carbon-settings';
+@import '../../global/styles/project-settings';
+
+@import 'carbon-components/scss/components/button/button';
+
+@mixin create-side-panel {
+  $block-class: #{$pkg-prefix}--create-side-panel;
+  $side-panel-block-class: #{$pkg-prefix}--side-panel__container;
+
+  // .#{$block-class}.#{$side-panel-block-class} {
+  //   box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.2), 0 2px 4px 0 rgba(0, 0, 0, 0.19);
+  // }
+
+  .#{$block-class}.#{$side-panel-block-class} .#{$block-class}__content-text {
+    padding-right: calc(20% - #{$spacing-05});
+  }
+
+  .#{$block-class}.#{$side-panel-block-class} .exp--side-panel__title-text {
+    margin-bottom: $spacing-02;
+    padding-right: calc(20% - #{$spacing-05});
+  }
+
+  .#{$block-class}.#{$side-panel-block-class} .exp--side-panel__subtitle-text {
```

Comment:
Gotta be careful here because the prefix could change at some point, swap out `.exp--side-panel` with `.#{$side-panel-block-class}.....`